File: Questions/Day-2/Cpp/Q2.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "Enter number: ";
    int num = 0;
    cin >> num;

    if (num % 2 == 0)
    {
        cout << "Even";
    }
    else
    {
        cout << "Odd";
    }

    return 0;
}

File: Questions/Day-2/Java/Q2.java
import java.util.Scanner;

public class Q2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Enter number: ");
        int num = in.nextInt();

        if (num % 2==0) {
            System.out.print("Even");
        } else {
            System.out.print("Odd");
        }
    }
}
